### ISSUE TITLE
Release version mixup

### ISSUE THREAD

**Issue by mdoucet** [zfit/phasespace#30]
Reviewing this for JOSS, I found that installing from the source code and installing with `pip install phasespace` produced different versions of the code, both with version number **1.0.2**.

Installing from pip gives me an older version of the code with the deprecated `phasespace.Particle` class instead of `phasespace.GenParticle`.

**Comment by apuignav** [comment]
@mayou36 do you know what may have happened?

**Comment by mayou36** [comment]
Yes, we usually kept the version on the development branch unchanged to the previous release and did not mark dev versions explicitly. I've added a 'dev0' to clarify the difference, so installing from the HEAD now yields a different version.

Out of interest: why did you install directly from github dev branch? I think for the review, the [release](https://github.com/zfit/phasespace/releases) is relevant, right?

**Comment by mdoucet** [comment]
Thanks for the info. I started out with the release version 1.0.2, and started following the documentation in your `README.rst` file (which we see when landing on your github project page). It seemed to make sense since it appeared to also be for version 1.0.2. When I started having problems I installed from the source code to investigate. I understand now, though. The `phasespace.readthedocs.io` page for 1.0.2 is also correct. You might want to label your version in the `README.rst` file, especially if there's a major API change.
